impl/src/test/groovy/com/okta/sdk/impl/util/OAuth2UtilsTest.groovy

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2014 Stormpath, Inc.
+ * Modifications Copyright 2018 Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.sdk.impl.util
+
+import com.okta.sdk.impl.config.ClientConfiguration
+import org.testng.annotations.Test
+
+import java.nio.file.Paths
+
+import static org.testng.Assert.assertNotNull
+
+class OAuth2UtilsTest {
+
+    @Test
+    void testCreateSignedJWT() {
+        ClientConfiguration clientConfiguration = new ClientConfiguration()
+        clientConfiguration.setClientId("client12345")
+        clientConfiguration.setKeyFilePath(Paths.get("src", "test", "resources", "privateKey.pem").toString())
+        clientConfiguration.setAlgorithm("RS256")
+        clientConfiguration.setBaseUrl("https://oktapreview-123.okta.com")
+        String signedJwt = OAuth2Utils.createSignedJWT(clientConfiguration)
+        assertNotNull(signedJwt)
+    }
+
```

Comment:
I suggest adding more tests to verify that you can create JWTs signed with both EC and RSA keys and verify the signature with the public key.

Reply:
word! will do.